### PR TITLE
Angela/reference update

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

**Review by soheiokamoto — APPROVED** [review]

File: products/logs/src/content/reference/glossary.md [hunk 1]
```diff
@@ -5,33 +5,35 @@ pcx-content-type: interim
...
 
 #### Origin requests
 
-Requests served from the origin server
+Requests served from the origin server.
 
 #### Cached requests
 
-The number of requests served from Cloudflare without having to hit the origin server. Cached requests are the sum of all requests where **CacheCacheStatus** equals _hit_, _stale_, _updating_, _ignored_. This doesn’t include _revalidated_ since the request had to be sent to the origin server.
+The number of requests served from Cloudflare without having to hit the origin server. Cached requests are the sum of all requests where **CacheCacheStatus** equals _hit_, _stale_, _updating_, _ignored_. This does not include _revalidated_ since the request had to be sent to the origin server.
```
Comment (marciocloudflare) [inline products/logs/src/content/reference/glossary.md:22]: I think terms like "_hit_", "_stale_", etc, should be monospaced. Italics are only for options customers can choose from, like in a drop-down menu

File: products/logs/src/content/reference/pathing-status.md [hunk 2]
```diff
@@ -74,35 +74,35 @@ Certain combinations of pathing have been labeled in the Cloudflare **Threat Ana
...
-| macro.chl.captchaErr | Captcha Error |
-| macro.chl.captchaFail | Human Challenged |
-| macro.chl.captchaNew | New CAPTCHA (CF) |
-| macro.chl.jschlFail | Browser Challenged |
-| macro.chl.jschlNew | Challenged threat |
-| macro.chl.jschlErr | Bot request |
-| user.chl.captchaNew | New CAPTCHA (user) |
+| `bic.ban.unknown` | Bad browser |
+| `hot.ban.unknown` | Blocked hotlink |
+| `hot.ban.ip` | |
```
Comment (marciocloudflare) [inline products/logs/src/content/reference/pathing-status.md:79]: This one has no label?

File: products/logs/src/content/reference/pathing-status.md [hunk 3]
```diff
@@ -159,82 +159,82 @@ The Cloudflare **Firewall Rules** app triggers actions based on matching custome
 
 | EdgePathingStatus | Description | EdgePathingOp | EdgePathingSrc | Status Code |
```
Comment (marciocloudflare) [inline products/logs/src/content/reference/pathing-status.md:160]: There's a status code field in this table, but no code. Do we need to have the status code?

File: products/logs/src/content/reference/pathing-status.md [hunk 4]
```diff
@@ -159,82 +159,82 @@ The Cloudflare **Firewall Rules** app triggers actions based on matching custome
...
 To understand the behavior of challenge pages, see [JavaScript and Captcha Challenge](#javascript-and-captcha-challenge).
 
 ## Zone Lockdown
 
-_Zone Lockdown_ blocks visitors to particular URIs where the visitor's IP is not allowlisted.
+**Zone Lockdown** blocks visitors to particular URIs where the visitor's IP is not allowlisted.
 
 <TableWrap>
 
 | EdgePathingStatus | Description | EdgePathingOp | EdgePathingSrc | Status Code |
```
Comment (marciocloudflare) [inline products/logs/src/content/reference/pathing-status.md:175]: Same question here for status code

File: products/logs/src/content/reference/pathing-status.md [hunk 5]
```diff
@@ -159,82 +159,82 @@ The Cloudflare **Firewall Rules** app triggers actions based on matching custome
...
 To understand the behavior of challenge pages, see [JavaScript and Captcha Challenge](#javascript-and-captcha-challenge).
 
 ## Firewall User-Agent Block
 
-Challenge (Captcha or JavaScript) or block visitors who use a browser for which the User-Agent name matches a specific string
+Challenge (Captcha or JavaScript) or block visitors who use a browser for which the User-Agent name matches a specific string.
 
 <TableWrap>
 
 | EdgePathingStatus | Description | EdgePathingOp | EdgePathingSrc | Status Code |
```
Comment (marciocloudflare) [inline products/logs/src/content/reference/pathing-status.md:189]: Another blank status code

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**